planner/create.go

```diff
@@ -135,24 +133,14 @@ func (n *createNode) Close() error {
 
 func (n *createNode) Source() planNode { return n.results }
 
-func (n *createNode) simpleExplain() (map[string]any, error) {
-	data := map[string]any{}
-	err := json.Unmarshal([]byte(n.newDocStr), &data)
-	if err != nil {
-		return nil, err
-	}
-
-	return map[string]any{
-		dataLabel: data,
-	}, nil
-}
-
 // Explain method returns a map containing all attributes of this node that
 // are to be explained, subscribes / opts-in this node to be an explainablePlanNode.
 func (n *createNode) Explain(explainType request.ExplainType) (map[string]any, error) {
 	switch explainType {
 	case request.SimpleExplain:
-		return n.simpleExplain()
+		return map[string]any{
+			dataLabel: n.input,
```

Comment:
todo: the name `dataLabel`doesn't make sense anymore. Please change to `inputLabel`